src/opm/output/eclipse/AggregateUDQData.cpp

```diff
@@ -420,8 +420,13 @@ namespace {
         {
             if (udq_input.is<Opm::UDQDefine>()) {
                 const auto& udq_define = udq_input.get<Opm::UDQDefine>();
+                const auto& update_status =  udq_define.status();
                 const auto& tokens = udq_define.tokens();
-                iUdq[0] = 2;
+                if (update_status.first == Opm::UDQUpdate::ON) {
```

Comment:
I had hoped that `IUDQ[0]` was the difference between UDQS which were defined and those that were only assigned - but that seems not to be the case any longer?